platform/android/java/editor/build.gradle

```diff
@@ -76,25 +76,86 @@ android {
         }
     }
 
+    flavorDimensions "xrMode", "xrRuntime"
+    productFlavors {
+        pancake {
+            dimension "xrMode"
+        }
+        openxr {
+            dimension "xrMode"
+            ndk {
+                abiFilters "arm64-v8a"
+            }
+            applicationIdSuffix ".openxr"
+            versionNameSuffix "-openxr"
+        }
+
+        google {
+            dimension "xrRuntime"
+        }
+        meta {
+            dimension "xrRuntime"
```

Comment:
Hmm, we need to add Khronos in here as well, still need to add the Khronos loader to the godot xr loader repo but I have no device to test this on.

Reply:
What device does the khronos loader run on?

The changes target the editor so if we don't plan to support/polish the experience on a particular set of devices, we omit the inclusion of their dependencies.

Reply:
HTC Focus I think, and on normal Android devices if you use Monado as an XR Runtime.